app/controllers/admin/owners_controller.rb

```diff
@@ -2,7 +2,7 @@ class Admin::OwnersController < ApplicationController
 
   def index
     if current_admin?
-      @owners = User.joins(:homes).uniq
```

Comment:
This Record Querying should go in a model, Going to move it to User model as a method to return users with homes